### ISSUE TITLE
Accessibility concerns for people who are blind and visually impaired.

### ISSUE THREAD

**Issue by ghost** [Engine-Simulator/engine-sim-community-edition#73]
Hello, my name is Luis, and I am a very big fan of this program. Unfortunately, there's an issue that I would like to try to address with the developer, And that is that screen reader support is virtually non-existent for people who are blind or visually impaired. I am completely blind and rely on a screen reader to use my computer effectively. Unfortunately, the user interface for Engine Sim is not accessible with a screen reader, I.e. NVDA, JAWS, etc.

For those who don't know what a screen reader is, in a nutshell, it is a software that allows people who are blind or visually impaired to see the screen using text-to-speech or braille. 

I am not exactly sure what this program is coded in since I'm not a programmer, but I would like to make some suggestions on accessibility improvements that could be made to the program to make it more accessible.

First things first, as a blind user of Engine Simulator, I am not able to load an engine independently, i.e., you have to use the mouse to click on the load engine button, which brings up the open dialog, which I am then able to navigate. But getting to the button requires sighted assistance. It would be helpful to have a keyboard shortcut that activates this function, such as Ctrl plus O for open or Ctrl plus L for load engine, etc. I've tried to change the engine using the MR file but that didn't seem to work. It seems to only let you load an engine using the load engine button within the program itself.

Another accessibility issue I've encountered is the status panel. I have no access to the panel to see, for example, whether an error has occurred in loading the engine script. I have to use my screen reader's OCR feature to try to read it, which works sometimes, but is a pain in the butt to do every single time an error happens. Speaking of the status panel, the console that it tells you to view if an error has occurred is not screen reader friendly either.

The program in general has no keyboard navigation, i.e. I can't press tab to move between elements within the user interface, such as the load engine button, the status panel, the console, and other areas of the program. Depending on what this is coded in, I'm not sure what can be done to fix this.
I'm not sure if I'm the only visually impaired user using this program, but if I am, I would love to help as much as I possibly can in guiding you in fixing these issues. Whether it's testing, giving feedback, etc. I would be happy to help fix the issues I've described above. This program is very fun to use. Especially with all the different engine sounds that you can hear. I'm hoping this gets somewhere, but we shall see.
